examples/autodiscover.cpp

```diff
@@ -34,14 +34,16 @@ int main()
         const auto env = ews::test::environment();
         ews::basic_credentials credentials(env.autodiscover_smtp_address,
                                            env.autodiscover_password);
+        ews::autodiscover_hints hints;
+        hints.autodiscover_url = "exch.otris.de";
 
         auto ews_url =
             ews::get_exchange_web_services_url<http_request>(
```

Comment:
The result is not a single URL anymore. So this should be named more like

```c++
auto result = /* ... */;
```

right? Remember, this is an example. So it should be as readable as possible for the end-user.